ui/components/src/model/units/bytes.ts

```diff
@@ -37,37 +38,52 @@ export const BYTES_UNIT_CONFIG: Readonly<Record<BytesUnitKind, UnitConfig>> = {
   },
 };
 
-export function formatBytes(bytes: number, { abbreviate, decimal_places }: BytesUnitOptions) {
-  if (bytes === 0) return '0 bytes';
+export function formatBytes(bytes: number, options: BytesUnitOptions) {
+  const { abbreviate, decimal_places } = options;
 
-  decimal_places = decimal_places ?? DEFAULT_DECIMAL_PLACES;
-  // Avoids maximumFractionDigits value is out of range error. Possible values are 0 to 20.
-  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#minimumfractiondigits
-  if (decimal_places < 0) {
```

Comment:
Moved this code to a function called `limitDecimalPlaces()`, and used it in all the formatting files.